app/services/tweet_stat_adjustment_service.rb

```diff
@@ -0,0 +1,46 @@
+class TweetStatAdjustmentService
+  ADVANCE_IN_HOURS = 8
+
+  def initialize(content)
+    @content = content
+  end
+
+  def call
+    tweet_nodes.each do |node|
+      adjust(node.tweet)
+    end
+  end
+
+  def tweet_nodes
```

Comment:
I believe this should be private, it's never called from the outside, just in tests...